db/migrate/20170530174228_add_cmd_to_problems.rb

```diff
@@ -0,0 +1,5 @@
+class AddCmdToProblems < ActiveRecord::Migration[5.1]
```

Comment:
Same as other one. Limiting the number to migrations will speed up `make deploy` too.